### PR TITLE
Release v0.0.1

### PR DESCRIPTION
**Pull request by eavgerinos**
This is the initial release of `ferto` gem.

### CODE REVIEW

File: spec/ferto/client_spec.rb [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        expect(subject.scheme).to eq default_scheme
+        expect(subject.host).to eq default_host
+        expect(subject.port).to eq default_port
+      end
+    end
+
+    context 'when we are passing different configuration options' do
+      subject { Ferto::Client.new(opts) }
+
+      let(:opts) { { host: 'foo-downloader.skroutz.gr' } }
```
Comment (agis) [inline spec/ferto/client_spec.rb]: `downloader.example.com`

File: spec/ferto/client_spec.rb [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+  describe '#download' do
+    subject { downloader.download(params) }
+
+    let(:params) do
+      {
+        aggr_id: 'bucket1',
+        aggr_limit: 3,
+        url: 'https://foo.bar/a.jpg',
+        callback_url: 'http://kavatza.skroutz.gr/downloads/myfile',
```
Comment (agis) [inline spec/ferto/client_spec.rb]: `example.com/downloads/myfile`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+# Ferto
+
+Ferto is a Ruby client for [skroutz/downloader](https://github.com/skroutz/downloader).
```
Comment (agis) [inline README.md]: > [...] a Ruby client for for downloader ...

File: lib/ferto/version.rb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+module Ferto
+  VERSION = "0.1.0"
```
Comment (konstantinoskostis) [inline lib/ferto/version.rb]: `%s/0.1.0/0.0.1`

Comment (konstantinoskostis) [reply lib/ferto/version.rb]: or change the commit message to `Release v0.1.0`

File: lib/ferto/callback.rb [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    end
+
+    # @return [Boolean] Whether the download job was successful.
+    def download_successful?
+      @success
+    end
+
+    private
+
+    # @raise [DownloaderApi::Callback::ParseError] if extra is not valid JSON
```
Comment (0xdiba) [inline lib/ferto/callback.rb]: `Ferto::Callback::ParseError`

File: lib/ferto/callback.rb [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+      {
+        success: params['success'],
+        error: error,
+        extra: extra,
+        download_url: params['download_url'],
+        resource_url: params['resource_url'],
+        job_id: params['job_id'],
+        response_code: params['response_code'],
+        # TODO extract this to a new field on the downloader side
```
Comment (0xdiba) [inline lib/ferto/callback.rb]: I think this TODO can go

**Pull request merged**